test/modellib/ModelLib.cpp

```diff
@@ -54,14 +54,41 @@ bool ModelLibBuilder::compileAndLoad(
   return compileAndLoad();
 }
 
+bool ModelLibBuilder::checkInstructionFromEnv() {
+  std::string instructionName =
+      getenv("TEST_CHECK_INSTRUCTION") ? getenv("TEST_CHECK_INSTRUCTION") : "";
+  if (instructionName.empty())
+    return true;
+  return checkInstruction(instructionName);
+}
+
+bool ModelLibBuilder::checkInstruction(const std::string instructionName) {
+  if (instructionName.empty())
+    return true;
+  std::string sharedLibName = getSharedLibName(sharedLibBaseName);
+  llvm::sys::DynamicLibrary sharedLibraryHandle =
```

Comment:
Please extract the library from the ExecutionSession model variable `onnx_mlir::ExecutionSession *exec;`

Reply:
Done. Thanks!